Sources/MuxUploadSDK/PublicAPI/Options/UploadOptions.swift

```diff
@@ -207,6 +207,33 @@ public struct UploadOptions {
         self.eventTracking = eventTracking
     }
 
+    /// - Parameters:
+    ///     - eventTracking: event tracking options for the
+    ///     direct upload
+    ///     - inputStandardization: options to enable or
+    ///     disable standardizing the format of the direct
+    ///     upload inputs, it is requested by default. To
+    ///     prevent the SDK from making any changes to the
+    ///     format of the input use ``UploadOptions.InputStandardization.skipped``
+    ///     - chunkSize: the size of each file chunk in
+    ///     bytes the SDK sends when uploading, default
+    ///     value is 8MB
+    ///     - retriesPerChunk: number of retry attempts
+    ///     if the chunk request fails, default value is 3
+    public init(
+        eventTracking: EventTracking = .default,
+        inputStandardization: InputStandardization = .default,
+        chunkSizeInBytes: Int = 8 * 1024 * 1024,
```

Comment:
Swift won't allow use of a constant as a default argument unless it is also made publicly accessible. Need to think through the right spelling for these before exposing so for now they're (sadly) repeated literals.